charts/registry-scanner/templates/configmap.yaml

```diff
@@ -13,7 +13,9 @@ data:
       url: {{ .Values.config.registryURL }}
       user: from-secret
       pass: from-secret
-      {{- if .Values.config.registryType }}
+      {{- if and .Values.config.registryType (eq .Values.config.registryType "ocp") }}
+      type: dockerv2
```

Comment:
will need to change this to `ocp` when PRs https://github.com/draios/registry-scanner/pull/287 and https://github.com/draios/ui/pull/5782 are merged

Reply:
or just remove whole section :)